bgpd/bgp_debug.c

```diff
@@ -916,15 +923,10 @@ DEFUN (no_debug_bgp_bestpath_prefix,
 	int idx_ipv4_ipv6_prefixlen = 4;
 	struct prefix *argv_p;
 	int found_prefix = 0;
-	int ret;
 
 	argv_p = prefix_new();
-	ret = str2prefix(argv[idx_ipv4_ipv6_prefixlen]->arg, argv_p);
-	if (!ret) {
-		prefix_free(argv_p);
-		vty_out(vty, "%% Malformed Prefix\n");
-		return CMD_WARNING_CONFIG_FAILED;
-	}
+	str2prefix(argv[idx_ipv4_ipv6_prefixlen]->arg, argv_p);
```

Comment:
Why ignore the return value of str2prefix()? Is it safe to assume that the CLI will handle all errors for us? I can already foresee Coverity Scan whining here.

Reply:
CLI will not allow the command to be entered if the input is not a valid prefix, as determined by the relevant matching function in `command_match.c`

Reply:
That's cool.

What about doing a `(void)str2prefix(...)` here then? This will make it explicit that we know what we're doing and keep Coverity Scan happy.

Reply:
Will make the `(void)str2prefix(...)` change everywhere in this file